File: go/chapter8/sandwiches.go
package main

import "fmt"

func main() {
	sandwichItems("cake", "sugar")
	sandwichItems("mint", "pepper", "cream", "milk")
	sandwichItems("chocolate")
}

func sandwichItems(items ...string) {
	fmt.Printf("The sandwich items:\n")

	for _, item := range items {
		fmt.Println(item)
	}
	fmt.Println()
}

File: javascript/chapter8/sandwiches.js
sandwichItems("cake", "sugar");
sandwichItems("mint", "pepper", "cream", "milk");
sandwichItems("chocolate");

/** @param {string[]} items */
function sandwichItems(...items) {
	console.log("The sandwich items:");

	for (const item of items) console.log(item);

	console.log();
}
